src/main.cpp

```diff
@@ -1,8 +1,18 @@
 #include <iostream>
-#include "CourseDashboard.hpp"
-
+#include "UserData.hpp"
+#include <unistd.h>
+#include "Group.hpp"
 int main()
 {
-    std::cout << "Course Dashboard" << std::endl;
+
+     UserData user("Krzysiek", "Cypher", "https://www.testowy-link.og", "https://www.testowy-link-22.og");
+    
+
+    sleep(1);
+    std::cout << "List of students: " << std::endl;
```

Comment:
You can make operator<< for UserData class (but you also need toString() method for enum class Group).
operator<<: https://medium.com/@nonuruzun/overloading-input-output-operators-in-c-a2a74c5dda8a
enum class toString - the easiest way is to use switch.